### PR TITLE
Update server-requirements.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected statement about additional NIC's

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ Recommended hardware for Edge Servers, standalone Mediation Servers, Video Inter
...
+|Network  <br/> |1 dual-port network adapter, 1 Gbps or higher (2 network adapters can be used, but they need to be teamed with a single MAC address and a single IP address).  <br/> Dual or multi-homed configurations are **not** supported for Video Interop Servers and Directors. <br/> Edge servers will require two network interfaces that are dual-port network adapters, 1 Gbps or higher (or two paired network adapters, for a total of four, each pair being teamed with a single MAC address and a single IP address, for a total of two pairs).  <br/> On standalone Mediation Servers the installation of additional network interface cards (NICs) to allow the configuration of a specific PSTN IP address is not supported.  <br/> |
```
Comment (nenonix) [inline Skype/SfbServer/plan-your-deployment/requirements-for-your-environment/server-requirements.md]:
```suggestion
|Network  <br/> |1 dual-port network adapter, 1 Gbps or higher (2 network adapters can be used, but they need to be teamed with a single MAC address and a single IP address).  <br/> Dual or multi-homed configurations are **not** supported for Video Interop Servers and Directors. <br/> Edge servers will require two network interfaces that are dual-port network adapters, 1 Gbps or higher (or two paired network adapters, for a total of four, each pair being teamed with a single MAC address and a single IP address, for a total of two pairs).  <br/> On standalone Mediation Servers the installation of additional network interface cards (NICs) to allow the configuration of a specific PSTN IP address is **not** supported.  <br/> |
```

**Comment by scanum** [comment]
@LanaChin Hi Lana, changes look correct. Could you please confirm and merge?

**Comment by LanaChin** [comment]
@SharS - over to you.

**Comment by SharS** [comment]
Reviewed by Dario Woitasen: dual NIC on Standalone Mediation Servers is supported since OCS 2007:
https://docs.microsoft.com/en-us/previous-versions/office/communications-server/ff518437(v=ocs.14)

**Pull request closed without merging**